src/script/lua_api/l_object.cpp

```diff
@@ -1244,6 +1244,21 @@ int ObjectRef::l_get_look_horizontal(lua_State *L)
 	return 1;
 }
 
+// get_look_roll(self)
+int ObjectRef::l_get_look_roll(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkObject<ObjectRef>(L, 1);
+	RemotePlayer *player = getplayer(ref);
+	if (player == nullptr)
+		return 0;
```

Comment:
What does returning a `0` indicate?

Reply:
The returned integer indicates the number of return values (as seen on the Lua side). So this means "return nothing", equivalent to just an early `return` in Lua.